Week3/homework/step3-bonus.js

```diff
@@ -3,12 +3,15 @@
 const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
 
 function makeUnique(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  arr = arr.filter((item, index, array) => array.indexOf(item) === index);
+  return arr;
 }
 
 const uniqueValues = makeUnique(values);
 console.log(uniqueValues);
+/* This can be du=one using Set to store unique values as:
+let uniqueValues = [...new Set(values)];
+*/
```

Comment:
👏 - I think using Sets here is a better idea than a `filter` and `indexOf` operation 🙂